utils/s3/client.cc

```diff
@@ -234,26 +235,84 @@ future<> map_s3_client_exception(std::exception_ptr ex) {
         auto e = std::current_exception();
         return make_exception_future<>(storage_io_error(EIO, format("S3 error ({})", e)));
     }
+}
+
+future<aws::aws_error> client::look_for_error(const http::reply& reply, input_stream<char>& in_, multipart_upload_completion is_mpu_completion) {
```

Comment:
Why did you move this method down the .cc file?

Reply:
Just for convenience, it was moved to where it is used so one can have it in the view, but you are right, this change is difficult to review, will move it back

Reply:
done